polaris/ocean/tasks/inertial_gravity_wave/__init__.py

```diff
@@ -8,4 +12,45 @@ def add_inertial_gravity_wave_tasks(component):
     component : polaris.ocean.Ocean
         the ocean component that the task will be added to
     """
-    component.add_task(Convergence(component=component))
+    component.add_task(InertialGravityWave(component=component))
+
+
+class InertialGravityWave(Task):
+    """
+    The convergence test case for inertial gravity waves
+    """
+
+    def __init__(self, component):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        component : polaris.ocean.Ocean
+            The ocean component that this task belongs to
+        """
+        name = 'inertial_gravity_wave'
+        subdir = f'planar/{name}'
+        super().__init__(component=component, name=name, subdir=subdir)
+
+        self.resolutions = [200., 100., 50., 25.]
```

Comment:
In a later PR, we should probably move these to being config options

Reply:
Yes, I agree.  I decided not to do that here but it should be changed.